app/helpers/events_helper.rb

```diff
@@ -0,0 +1,11 @@
+module EventsHelper
+  def event_params
```

Comment:
- [ ] If a method is only needed in the controller it should be a private method in its controller.
The methods in the helpers are the ones you need to access in the views and should not connect directly to model methods. EX(Post.new, Comment.create, current_user.friends.find_by(...)...)

- Check the link in the previous review about where you should put your code in rails to keep the project tidy.